### ISSUE TITLE
Dropdown component aria-selected not announced to screen reader users

### ISSUE THREAD

**Issue by dnvargas25** [wwnorton/design-system#14]
In the Storybook, when a user selects an option from the dropdown component, a checkmark appears next to the option and the `aria-selected="true"` attribute is applied to it. 

When testing the dropdown with the arrows keys and the options open/visible:
- NVDA in Firefox/Chrome, JAWS in Chrome, and VoiceOver in Safari/Chrome did not note that an option was "selected" (or "unselected"). 
- NVDA in Edge said that other, non-selected options were "unselected."
- JAWS in Firefox said that all options were "selected." 

All screen readers noted which option was selected when the dropdown was closed though. Is this the intended experience for `aria-selected` or should options be read aloud as "selected"/"unselected" when arrowing through them?

**Comment by sh0ji** [comment]
This should be following [the APG Listbox pattern](https://w3c.github.io/aria-practices/#Listbox) quite closely. Do you observe the same issues in [the Collapsible Dropdown example](https://w3c.github.io/aria-practices/examples/listbox/listbox-collapsible.html)?

**Comment by dnvargas25** [comment]
I do observe the same issues in the Collapsible Dropdown example. I think my expectation for that example was a little different because, visually, the checkmark followed my focus as I arrowed through the options. In the Storybook example, the checkmark stays on the previously selected item instead of following focus through the list, so when I re-focused on the previously selected (checked) item, I was wondering if there should be some indication that it was selected. But I understand that this isn't necessary--I was hung up with a visual element that doesn't impact the usability of the dropdown. Thanks for explaining!
